vendor.conf

```diff
@@ -17,8 +17,9 @@ github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/docker/notary v0.4.2
 github.com/docker/swarmkit b19d028de0a6e9ca281afeb76cea2544b9edd839
 github.com/flynn-archive/go-shlex 3f9db97f856818214da2e1057f8ad84803971cff
-github.com/golang/protobuf 8ee79997227bf9b34611aee7946ae64735e6fd93
+github.com/go-check/check 4ed411733c5785b40214c70bce814c3a3a689609 https://github.com/cpuguy83/check.git
```

Comment:
For now it is required. There is an import from `docker/docker/pkg/testutil/cmd` because some people insisted that I support the matcher interface, which required that import.

`pkg/testutil/` imports `pkg/testutil/cmd` and this repo uses `pkg/testutil`.

We can definitely fix this in two ways. 

1. remove the matcher from `pkg/testutil/cmd`, I don't think it's actually used much, we mostly do `result.Assert()` everywhere.
2. split `pkg/testutil/` so that it doesn't have things for both integration testing and unit testing.

I think we should do both.